applications/tari_base_node/src/builder.rs

```diff
@@ -905,14 +905,23 @@ async fn setup_wallet_comms(
 }
 
 async fn add_peers_to_comms(comms: &CommsNode, peers: Vec<Peer>) -> Result<(), String> {
+    let node_identity = comms.node_identity().clone();
```

Comment:
No need to clone here - a reference should do

Reply:
Think the value will be freed after the call, will add it directly to the comparison

Reply:
Nope :)